tests/lib/outputs/compliance/generic/generic_aws_test.py

```diff
@@ -0,0 +1,76 @@
+from datetime import datetime
+from io import StringIO
+
+from freezegun import freeze_time
+from mock import patch
+
+from prowler.lib.outputs.compliance.generic.generic import GenericCompliance
+from prowler.lib.outputs.compliance.generic.models import Generic
+from tests.lib.outputs.compliance.fixtures import NIST_800_53_REVISION_4_AWS
+from tests.lib.outputs.fixtures.fixtures import generate_finding_output
+from tests.providers.aws.utils import AWS_ACCOUNT_NUMBER, AWS_REGION_EU_WEST_1
+
+
+class TestAWSENS:
```

Comment:
Review this class name please.
```suggestion
class TestAWSGenericCompliance:
```